examples/gno.land/r/demo/boards2/acl.gno

```diff
@@ -0,0 +1,98 @@
+package boards2
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/boards2/admindao"
+)
+
+const (
+	RoleOwner     Role = "owner"
+	RoleAdmin          = "admin"
+	RoleModerator      = "moderator"
+)
+
+type (
+	// Role defines the type for user roles.
+	Role string
+
+	// ACL or access control list manages user roles and permissions.
+	ACL struct {
```

Comment:
This type could be named `Permissions` (or `DefaultPermissions) instead of `ACL`, naming it `ACL` makes sense thinking that it not only handles permissions but also roles.